scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/mini-circuits.yml

```diff
@@ -0,0 +1,27 @@
+Mini-Circuits_DB1627:
+  device_type: DFN
+  library: RF_Mini-Circuits
+  manufacturer: Mini-Circuits
+  custom_name_format: "{man}_{mpn}"
+  part_number: DB1627
+  size_source: https://ww2.minicircuits.com/case_style/DB1627.pdf
+  ipc_class: dfn
+  body_size_x:
+    nominal: 3.81
+    tolerance: 0.127
```

Comment:
Isn't the +/-0.01 inch in the datasheet translated into 0.254 mm here? I think your tolerance is half the actual value.

Reply:
The tolerance should be correct. The tolerances are for the values in inches. In inches it has 3 places, so it is +-0.005 inch.